src/DssSpell.t.sol

```diff
@@ -878,5 +930,141 @@ contract DssSpellTest is DssSpellTestBase {
     }
+
+    function testChainLinkPaymentAdapterTreasury() public {
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        require(NetworkPaymentAdapterLike(wallets.addr("CHAINLINK_PAYMENT_ADAPTER")).treasury() == wallets.addr("CHAINLINK_TREASURY"), "ChainLink/incorrect-treasury");
+    }
+
+    string OLD_RWA007_DOC      = "QmejL1CKKN5vCwp9QD1gebnnAM2MJSt9XbF64uy4ptkJtR";
+    string NEW_RWA007_DOC      = "QmY185L4tuxFkpSQ33cPHUHSNpwy8V6TMXbXvtVraxXtb5";
+
+    function testRWA007DocChange() public {
+        _checkRWADocUpdate("RWA007-A", OLD_RWA007_DOC, NEW_RWA007_DOC);
+    }
+
+    // RWA tests
+
+    address RWA015_A_OPERATOR = addr.addr("RWA015_A_OPERATOR");
+    address RWA015_A_CUSTODY  = addr.addr("RWA015_A_CUSTODY");
+
+    RwaUrnLike               rwa015AUrn             = RwaUrnLike(addr.addr("RWA015_A_URN"));
+    RwaOutputConduitLike     rwa015AOutputConduit   = RwaOutputConduitLike(addr.addr("RWA015_A_OUTPUT_CONDUIT"));
+    RwaLiquidationOracleLike oracle                 = RwaLiquidationOracleLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
+    GemAbstract              psmGem                 = GemAbstract(rwa015AOutputConduit.gem());
+
+    uint256 daiPsmGemDiffDecimals                   = 10 ** (dai.decimals() - psmGem.decimals());
+
+    function testRWA015_OUTPUT_CONDUIT_DEPLOYMENT_SETUP() public {
+        assertEq(rwa015AOutputConduit.dai(), addr.addr("MCD_DAI"),       "output-conduit-dai-not-match");
+        assertEq(rwa015AOutputConduit.gem(), addr.addr("PAXUSD"),        "output-conduit-gem-not-match");
+        assertEq(rwa015AOutputConduit.psm(), addr.addr("MCD_PSM_PAX_A"), "output-conduit-psm-not-match");
+    }
+
+    function testRWA015_INTEGRATION_CONDUITS_SETUP() public {
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        assertEq(rwa015AUrn.outputConduit(), address(rwa015AOutputConduit), "RwaUrn/urn-outputconduit-not-match");
+
+        assertEq(rwa015AOutputConduit.wards(pauseProxy),      1, "OutputConduit/ward-pause-proxy-not-set");
+        assertEq(rwa015AOutputConduit.wards(address(esm)),    1, "OutputConduit/ward-esm-not-set");
+        assertEq(rwa015AOutputConduit.can(pauseProxy),        0, "OutputConduit/pause-proxy-hoped");
+        assertEq(rwa015AOutputConduit.can(RWA015_A_OPERATOR), 1, "OutputConduit/operator-not-hope");
+        assertEq(rwa015AOutputConduit.may(pauseProxy),        0, "OutputConduit/pause-proxy-mated");
+        assertEq(rwa015AOutputConduit.may(RWA015_A_OPERATOR), 1, "OutputConduit/operator-not-mate");
+        assertEq(rwa015AOutputConduit.bud(RWA015_A_CUSTODY),  1, "OutputConduit/destination-address-not-whitelisted-for-pick");
+
+        assertEq(rwa015AOutputConduit.quitTo(), address(rwa015AUrn), "OutputConduit/quit-to-not-urn");
+    }
+
+    function testRWA015_OPERATOR_DRAW_CONDUIT_PUSH() public {
```

Comment:
The current tests seem to be "not more, not less than needed".
I would still be more comfortable with the full set of tests added - i.e.:
  - "excessive, but can't hurt"
  - does not break the pattern of rwa-related spell actions.

If it's not a big deal - i would ask to add them anyway.

Reply:
I don’t see any reason of testing repaying debt as it’s use another conduit which was tested when this ilk was on onboarded. Also this will not give us any additional coverage (for me it’s looks like more as waste of time). It’s not a matter of copy pasting code because we are paying in different token (usdc), so this test should be adjusted and I don’t see a reason to spend time to it if it doesn’t help us to ensure security